src/Uart/Uart.c

```diff
@@ -53,6 +53,38 @@ getAddressBase(Uart_Id id)
     }
 }
 
+static inline bool
+setBaudRate(const Uart* const uart, const Uart_BaudRate baud)
+{
+    uint32_t baudValue = 0;
+    switch(baud) {
+        case 300:
+        case 600:
+        case 1200:
```

Comment:
Also, this could be extracted to convert baud rate, or something like that. Not required though.

Reply:
done